frontend/app/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -18,7 +18,7 @@ import {
   useAverageInterestRate,
   useBranchDebt,
   useEarnPool,
-} from "@/src/liquity-utils";
+} from '@/src/liquity-utils';
```

Comment:
We use `dprint` for autoformatting, although it's not enforced by CI. If possible, try to setup your editor to use it for formatting. Otherwise, it can be run manually via `pnpm dprint fmt`.